packages/ui-axe-check/src/runAxeCheck.ts

```diff
@@ -21,30 +21,39 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
 import axeCore from 'axe-core'
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'element' implicitly has an 'any' type.
-export default async function runAxe(element, options = {}) {
-  let result = true
-  const context = {
+type AxeCheckOptions = {
+  exclude?: string | string[]
+  /**
+   * Violations to ignore/filter out. 'color-contrast' is automatically excluded
+   * because we test color contrast in theme tests.
+   */
+  ignores?: string[]
+}
+
+export default async function runAxe(
+  element: string,
+  options: AxeCheckOptions = {}
+) {
+  let result: Error | true = true
+  const context: axeCore.ContextObject = {
     include: [element],
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'exclude' does not exist on type '{}'.
     exclude: options.exclude
   }
   const config = {
```

Comment:
I'd rather do this than a cast:
```suggestion
  const config: axeCore.RunOptions = {
```

Reply:
fixed